src/components/BoxCard.jsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-unused-vars */
+import { Transition } from "@headlessui/react";
+import { useState, useEffect } from "react"
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import Settings from "../images/Settings";
+import ViviHourglass from "../images/ViviHourglass"
+import { ChromePicker } from "react-color"
+import { useFormik } from "formik";
+import { toast } from "react-toastify";
+const BoxCard = ({ data }) => {
+    const [color, setColor] = useState("#1A1F32");
```

Comment:
not a huge fan, is there an another way to dot this ?

Reply:
not if we want to use tailwind and take advantage of its way to purge css files

Reply:
ok, resolve